src/components/listItems/ChallengeCard.tsx

```diff
@@ -0,0 +1,61 @@
+import { View, Text, TouchableOpacity, StyleSheet } from "react-native";
+import { useRouter } from "expo-router";
+
+import ProgressBar from "@src/components/ui/ProgressBar";
+
+const ChallengeCard = (props: any) => {
+  const router = useRouter();
+  const { challenge, isOngoing } = props;
+
+  const showDetails = () => {
+    router.navigate({
+      pathname: "challengeDetailsScreen",
+      params: {
+        name: challenge.name,
```

Comment:
This is good for now, but we should be passing the challenge id, and request the challenge detail in the Detail Page

Reply:
As discussed offline, this will be implemented in the next PR. For now, I added a TODO.